gateway/auth_manager.go

```diff
@@ -1,11 +1,14 @@
 package gateway
 
 import (
+	"context"
 	"encoding/base64"
 	"encoding/json"
 	"strings"
 	"time"
 
+	"github.com/TykTechnologies/tyk/internal/redis"
+
```

Comment:
reorder the import together with other TykTechnologies imports (move down about 4 lines)